['/home/z/build/1packages/chromium-dev-git/fetch_gclient_base/checkout_root/src/buildtools/linux64/gn', 'gen', '/tmp/out/Default']
ERROR at //content/public/child/BUILD.gn:20:9: Can't load input file.
        "//content/ppapi_plugin",
        ^-----------------------
Unable to load:
  /home/z/build/1packages/chromium-dev-git/fetch_gclient_base/checkout_root/src/content/ppapi_plugin/BUILD.gn
I also checked in the secondary tree for:
  /home/z/build/1packages/chromium-dev-git/fetch_gclient_base/checkout_root/src/build/secondary/content/ppapi_plugin/BUILD.gn
==> ERROR: A failure occurred in prepare().
    Aborting...

diff --git a/content/public/child/BUILD.gn b/content/public/child/BUILD.gn
index 00bdfa3..09f186f 100644
--- a/content/public/child/BUILD.gn
+++ b/content/public/child/BUILD.gn
@@ -17,7 +17,7 @@ group("child") {
     if (!is_android) {
       public_deps += [
         "//content/gpu",
-        "//content/ppapi_plugin",
+#        "//content/ppapi_plugin",
       ]
     }
   }
